CHANGELOG.md

```diff
@@ -851,17 +938,8 @@ replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8
 
 ### API Breaking Changes
 
-* (store) [#13516](https://github.com/cosmos/cosmos-sdk/pull/13516) Update State Streaming APIs:
```

Comment:
We should not remove this from a release

Reply:
this is included in 0.46, is there a reason we need to include is twice?

Reply:
Yes, because it is included only from that release in v0.46.
I think on v0.46 or v0.45 won't look at the changelog of other versions to know what is potentially included in their point version.

Same reasoning for the others.

Reply:
if someone is updating from 0.45 -> 0.47 they would most likely be reading the entire changelog from then. Ill add it back since it doesnt make a difference, just duplicates